src/puppetlabs/puppetdb/pdb_routing.clj

```diff
@@ -40,7 +41,8 @@
            "/cmd" (cmd/command-app get-shared-globals
                                    enqueue-raw-command-fn get-response-pub)
            "/query" (server/build-app get-shared-globals)
-           "/admin" (admin/build-app enqueue-command-fn query-fn)]))))
+           "/admin" (admin/build-app enqueue-command-fn query-fn)
+           (route/not-found "Not Found")]))))
```

Comment:
seems like we have a separate top-level route thing for PE. Does this need to be done over there as well?